HLTrigger/Muon/plugins/HLTScoutingTrackProducer.cc

```diff
@@ -41,13 +43,20 @@ class HLTScoutingTrackProducer : public edm::global::EDProducer<> {
   void produce(edm::StreamID sid, edm::Event& iEvent, edm::EventSetup const& setup) const final;
 
   const edm::EDGetTokenT<reco::TrackCollection> otherTrackCollection_;
+  const edm::EDGetTokenT<reco::VertexCollection> vertexCollection_;
+
+  const int mantissaPrecision;
+  const int vtxMinDist;
```

Comment:
```suggestion
  const float vtxMinDist;
```

Reply:
ah here I now took "const double vtxMinDist", following same type for "vtxMinDist" in Line 59 and 144. Shall I take float?

Reply:
I think the difference is negligible.